diff --git a/localstack/services/awslambda/lambda_executors.py b/localstack/services/awslambda/lambda_executors.py
index 19f1e07..85d4781 100644
--- a/localstack/services/awslambda/lambda_executors.py
+++ b/localstack/services/awslambda/lambda_executors.py
@@ -301,7 +301,7 @@ class LambdaExecutorReuseContainers(LambdaExecutorContainers):
     def startup(self):
         self.cleanup()
         # start a process to remove idle containers
-        self.start_idle_container_destroyer_interval()
+        # self.start_idle_container_destroyer_interval()
 
     def cleanup(self, arn=None):
         if arn:
